.gitignore

```diff
@@ -2,3 +2,5 @@ test/*
 !test/*.c
 ui/*
 !ui/*.c
+.idea
```

Comment:
This is an IDE-specific file type, and does not really belong in the repository; this repository should be agnostic to these kind of configurations.